File: Claude 3 Haiku/C++/2-3.c
#include <iostream>
#include <fstream>
#include <string>
using namespace std;

int main() {
    // Open the input file
    ifstream inputFile("input.txt");
    if (!inputFile.is_open()) {
        cout << "Unable to open input file." << endl;
        return 1;
    }

    // Open the output file
    ofstream outputFile("output.txt");
    if (!outputFile.is_open()) {
        cout << "Unable to open output file." << endl;
        inputFile.close();
        return 1;
    }

    // Read the content from the input file and write it to the output file
    string line;
    while (getline(inputFile, line)) {
        outputFile << line << endl;
    }

    // Close the files
    inputFile.close();
    outputFile.close();

    cout << "File read and write operations completed successfully." << endl;

    return 0;
}

File: Claude 3 Haiku/python/2-3.py
def read_write_files():
    try:
        # Read content from the input file
        with open("input.txt", "r") as input_file:
            input_content = input_file.read()

        # Write content to the output file
        with open("output.txt", "w") as output_file:
            output_file.write(input_content)

        print("File read and write operations completed successfully.")
    except IOError:
        print("An error occurred while accessing the files.")

if __name__ == "__main__":
    read_write_files()

